recipe/meta.yaml

```diff
@@ -134,6 +134,8 @@ test:
         - test -f $PREFIX/lib/libopencv_bioinspired${SHLIB_EXT}  # [unix]
         - test -f $PREFIX/lib/libopencv_hdf${SHLIB_EXT}          # [unix]
         - test -f $PREFIX/lib/libopencv_freetype${SHLIB_EXT}     # [unix]
+        - python -c "import cv2; 'Unknown' not in cv2.videoio_registry.getBackendName(cv2.CAP_V4L)"  # [unix]
```

Comment:
Shouldn't it be `[linux]` (137) and `[not linux]` (138)?

Reply:
it should be `[linux]`

I'm not sure if that line will cause failure. I've done something like that before, but I want to "test the test" with the line below it. I'll remove the line below it when the "test works".

Lets let this build complete, OSX should fail (because of my bad selecctor) but we can fix that readily.